.github/ISSUE_TEMPLATE/bug.md

```diff
@@ -0,0 +1,27 @@
+---
+labels: bug
+name: "Bug Report"
+about: "Report an error condition or unexpected behavior"
+---
+<!--- Provide a general summary of the issue in the Title above -->
+
+## Expected Behavior
+<!--- Explain what should happen -->
+
+## Current Behavior
+<!--- Describe what happens instead of the expected behavior -->
+
+## Steps to Reproduce
+<!--- Provide a link to a live example, or an unambiguous set of steps to -->
+<!--- reproduce this bug. Include code to reproduce, if relevant -->
+1.
+2.
+3.
+4.
+<!--- delete any unused steps -->
+
+## Possible Solution
+<!--- Do you have any ideas of how to implement the addition or change? -->
```

Comment:
I think this is used only for bugs , so "Do you have any ideas how to fix this bug" ?